src/components/LandDetailPage/UnsetENSContentModal.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { T, t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { Button, Modal } from 'decentraland-ui'
+import { ENS } from 'modules/ens/types'
+import { Land } from 'modules/land/types'
+import { Link } from 'react-router-dom'
+import { locations } from 'routing/locations'
+
+type UnsetENSContentModalProps = {
+  land: Land
+  ens: ENS
+  open: boolean
+  onCancel: () => void
+  onConfirm: () => void
+}
+
+const UnsetENSContentModal = ({ land, ens, open, onCancel, onConfirm }: UnsetENSContentModalProps) => (
+  <Modal size="tiny" open={open}>
+    <Modal.Header>{t('land_detail_page.clear_ens_content.title')}</Modal.Header>
```

Comment:
I'm nitpicking here but the component is named *Unset*ENSContent and the i18n key is  *clear*_ens_content
maybe we can use the same word?

Reply:
Updated